stories/loading.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { checkA11y } from 'storybook-addon-a11y';
+import { withInfo } from '@storybook/addon-info';
+import { LoadingMolecule } from '../components';
+
+const basePath = window.location.pathname + window.location.search;
+
+storiesOf('Loading molecules', module)
+  .addDecorator((story, context) => withInfo('common info')(story)(context))
+  .addDecorator(checkA11y)
+  .add('color', () => (
+    <div style={{ height: '300px' }}>
+      <LoadingMolecule basePath={basePath} startColor="#00ACA9" stopColor="#1F7F79" />
+    </div>
+  ))
+  .add('grayscale', () => (
+    <div style={{ height: '300px' }}>
+      <LoadingMolecule basePath={basePath} startColor="#BABABA" stopColor="#DADADA" />
+    </div>
+  ))
+  .add('small', () => (
+    <div style={{ height: '300px' }}>
+      <LoadingMolecule basePath={basePath} startColor="#00ACA9" stopColor="#1F7F79" type="small" />
+    </div>
+  ))
+  .add('large', () => (
+    <div style={{ height: '300px' }}>
+      <LoadingMolecule basePath={basePath} startColor="#00ACA9" stopColor="#1F7F79" type="large" />
+    </div>
+  ));
```

Comment:
It would make more sense if the filename would reflect the component name more, so in thise case rename this file to `loadingMolecule.js`.

Reply:
Also not rendering correctly because `basePath` returns something extremely exotic 😄